css/css-values/getComputedStyle-calc-mixed-units-003.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Values Test: computed value of calc() values using multiplication/division with mixed units</title>
```

Comment:
This is not using divisions. Wasn't that the point of this test?

Reply:
Yup, looking back at them now, those tests were not great.

Resolved in [b5749cf](https://github.com/web-platform-tests/wpt/pull/42517/commits/b5749cf3bcc1fee5712fe0986ea5bf946059e38c)

Reply:
Mind checking my work there?

Reply:
Looks good!